fix: intro (#191)
solana-foundation/developer-content: docs/intro/index.md

## What is Solana?

Solana is a blockchain built for mass adoption. It's a high performance network
that is utilized for a range of use cases, including finance, payments, and
gaming. Solana operates as a single global state machine, and is open,
interoperable, and decentralized.

## Why Solana?

Designed to scale, Solana is purpose built for blockchain applications to reach
millions of users. Instead of worrying about optimizing for the blockchain
layer, developers can focus on building their application to reach product
market fit. Not only can it scale now for blockchain application needs, but the
network continues to be optimized with user experience in mind.

Building the best user experience in an application is a top priority for
developers. With blockchains, the user experience is often limited by the

> underlying technology, causing slow response times and high fees. The low fees

Reviewer: ```suggestion
underlying technology, causing slow response times and high fees. Solana's low fees
```

and 400ms confirmation times enable developers to build applications that are
user friendly and accessible to everyone.

## Solana Features

| Feature                      | Description                                                                                                                                                                                                                  |
| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | On-chain Program Development | The ability of developing and deploying programs on-chain. Users can interact with these programs permissionlessly, removing the need for middleman servers to build applications.                                           |

Reviewer: something about the current working just seems off to me. I can't place exactly why though:

```suggestion
| On-chain Program Development | The ability to develop and deploy programs on-chain. Users can interact with these programs permissionlessly, removing the need for middleman servers to build applications.                                           |
```

> | 400 ms Slot Times            | Every transaction sent by users on Solana is confirmed in a block. With a target slot time of 400ms for each block interactions by users do not have to wait, providing the best in class user experience of any blockchain. |

Reviewer: ```suggestion
| 400ms Slot Times            | Every transaction sent by users on Solana is confirmed in a block. With a target slot time of 400ms for each block, interactions by users do not have to wait, providing the best in class user experience of any blockchain. |
```

> | Low Fees                     | Fees on Solana are well known to be low. With a 0.00064 median fee per transaction, Solana enables building for users across the world regardless of background.                                                             |

Reviewer: ```suggestion
| Low Fees                     | Fees on Solana are well known to be low. With a median fee of 0.00064 SOL per transaction, Solana enables building for users across the world regardless of background.                                                             |
```
